modules/virt-configuring-disk-sharing-lun.adoc

```diff
@@ -6,11 +6,12 @@
 [id="virt-configuring-disk-sharing-lun_{context}"]
 = Configuring disk sharing by using LUN
 
-You can configure a LUN-backed virtual machine (VM) disk to be shared among multiple virtual machines by enabling SCSI persistent reservation. Enabling the shared option allows you to use advanced SCSI commands, such as those required for a Windows failover clustering implementation, against the underlying storage. Any disk to be shared must be in block mode.
+To protect data on your VM from outside access, you can enable SCSI persistent reservation and configure a LUN-backed virtual machine disk to be shared among multiple virtual machines. Enabling the shared option allows you to use advanced SCSI commands, such as those required for a Windows failover clustering implementation, for managing the underlying storage.
 
-A disk of type `LUN` exposes the volume as a LUN device to the VM. This allows the VM to execute arbitrary iSCSI command passthrough on the disk.
 
-You reserve a LUN through the SCSI persistent reserve options to protect data on the VM from outside access. To enable the reservation, you configure the feature gate option. You then activate the option on the LUN disk to issue SCSI device-specific input and output controls (IOCTLs) that the VM requires.
+When a storage volume is configured as the `LUN` disk type, a VM can use the volume as a Logical Unit Number (LUN) device. As a result, the VM can deploy and manage the disk by using SCSI commands.
```

Comment:
```suggestion
When a storage volume is configured as the `LUN` disk type, a VM can use the volume as a logical unit number (LUN) device. As a result, the VM can deploy and manage the disk by using the SCSI commands.
```

Reply:
Is there a specific set of SCSI commands that the doc is talking about here? If not, then I don't think the definite article is really warranted here. On the other hand, you're right that capitalizing LUN is probably unnecessary.